lib/rubocop/cop/rails/i18n_locale_texts.rb

```diff
@@ -0,0 +1,109 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # Enforces use of I18n and locale files instead of locale specific strings.
+      #
+      # @example
+      #   # bad
+      #   class Product < ApplicationRecord
+      #     validates :code, format: { with: /\A[a-zA-Z]+\z/, message: "only letters" }
+      #   end
+      #
+      #   # good
+      #   # config/locales/en.yml
+      #   # en:
+      #   #   activerecord:
+      #   #     errors:
+      #   #       models:
+      #   #         product:
+      #   #           code_format: "only letters"
```

Comment:
Cosmetic: does this need to be double commented? YAML is a file, too, just like Ruby code below. Is there a specific semantic to it, will it appear differently in cop's docs?

Reply:
Unfortunately, this is required, because otherwise documentation generator raises an error regarding colons.

```
lib/rubocop/cop/rails/i18n_locale_texts.rb: Syntax Error in an example. unexpected token tCOLON
```